README.md

```diff
@@ -150,18 +146,33 @@ wsk action create iot-pub iot-pub.py
 
 Once the actions are successfully created, we can set default service credentials for each of the actions. Otherwise we’d have to pass in the service credentials every time we’d like our actions to call the Watson services. To obtain these credentials, click each provisioned service in the IBM Cloud dashboard, and then select the `View credentials` dropdown.
 
-<p align="center">
-<img src="/images/stt_creds.png" data-canonical-src="/images/stt_creds.png" width="600" height="400" style="margin-left: auto; margin-right: auto;" />
-</p>
+![](images/stt_creds.png)
 
 Then insert the corresponding credentials when running the commands below.
 
+* If the service credentials from IBM Watson Assistant is username/password based as below populate the username, password and workspace_id and comment out the IAM credentials part.
```

Comment:
Maybe change this to something along the lines of:
"If the service credentials from IBM Watson Assistant are username/password as shown below, populate the username, password, and workspace_id fields, and comment out the IAM credentials field."